src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -131,3 +140,10 @@ def load_command_table(self, _):
     # AKS egress commands
     with self.command_group('aks egress-endpoints', managed_clusters_sdk, client_factory=cf_managed_clusters) as g:
         g.custom_command('list', 'aks_egress_endpoints_list')
+
+    # AKS snapshot commands
+    with self.command_group('aks snapshot', snapshot_sdk, client_factory=cf_snapshots) as g:
```

Comment:
I think snapshot_sdk is already include client_factory in line 56.

Reply:
This follows the same patterns for other AKS resource type (e.g. managed_cluster in line 27 and line 60)